app/views/items/index.html.erb

```diff
@@ -1,36 +1,55 @@
 <% provide(:title, "Items") %>
+<% params[:page_size] ||= '10' %>
```

Comment:
Before I accept this pull request. I want to get confirmation that you've tested the pagination on this thing.